contrib/multivariate-tada/genunivdata.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import torch
+
+#Simulates univariate data based on demo mutation rates in original TADA code
+
+def genUnivData(gamma, q, N_family, N_case, N_control, mut_1, mut_2):
+    #Rates for poisson distributions that the counts follow
+
+    X_d1_rates = 2 * mut_1 * gamma * N_family
+    X_d2_rates = 2 * mut_2 * gamma * N_family
+    X_ctrl_rates = 5000 * [q * N_control]
+    X_case_rates = 5000 * [q * gamma * N_case]
+    X_t_rates = 5000 * [q * gamma * N_family]
+    X_nt_rates = 5000 * [q * N_family]
+
+    #Stacks poisson rates and samples
+    cls1 = torch.stack([X_d1_rates,torch.tensor(X_t_rates),torch.tensor(X_nt_rates),torch.tensor(X_case_rates),torch.tensor(X_ctrl_rates)])
+    cls2 = torch.stack([X_d2_rates,torch.tensor(X_t_rates),torch.tensor(X_nt_rates),torch.tensor(X_case_rates),torch.tensor(X_ctrl_rates)])
+    sample_1 = torch.poisson(cls1)
+    sample_2 = torch.poisson(cls2)
+
+    #Combines simulated data with mutation rates into table
+    table = torch.transpose(torch.cat((mut_1.unsqueeze(0),sample_1,mut_2.unsqueeze(0),sample_2),dim=0),0,1)
```

Comment:
What's happening here?  Do any of these intermediate expressions have domain-level interpretations that we could pull out and name?